source/tests/pt/test_tabulate.py

```diff
@@ -0,0 +1,126 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import unittest
+
+import numpy as np
+import torch
+
+from deepmd.pt.utils.tabulate import (
+    unaggregated_dy2_dx,
+    unaggregated_dy2_dx_s,
+    unaggregated_dy_dx,
+    unaggregated_dy_dx_s,
+)
+from deepmd.tf.env import (
+    op_module,
+    tf,
+)
+
+tf.compat.v1.enable_eager_execution()
```

Comment:
The eager execution should be disabled after this test. You may consider using `setUpModule` and `tearDownModule`.